jenkins/slave-base/Dockerfile.centos7

```diff
@@ -1,4 +1,4 @@
-FROM openshift/jenkins-slave-base-centos7
+FROM openshift/jenkins-slave-base-centos7:v3.11
```

Comment:
I don't think we should do this. It basically means we only support 3.11. See how we did this in #91.

Reply:
I will take a look.

Reply:
Reworked it, accordingly to #91